### ISSUE TITLE
Recording for April 12th

### ISSUE THREAD

**Issue by Andrea-Havron** [James-Thorson/2016_Spatio-temporal_models#12]
We you able to get a recording for April 12th's lecture? I see Week 3 Lab on the drive, but not Week 3 Lecture.

Thanks,
Andrea

**Comment by James-Thorson** [comment]
Nope, IT problems with recording that day, sorry. Should work from here out though
